lib/carin_for_blue_button_test_kit/client/v2.0.0/c4bb_client_test_suite.rb

```diff
@@ -106,41 +104,104 @@ def self.test_resumes?(test)
     group do
       run_as_group
       title 'Wait for Claims Data and Search Requests'
+      description %(
+        This test will wait for the client system under test to submit requests for resources for each of the
+        [CARIN for Blue Button profiles](https://hl7.org/fhir/us/carin-bb/STU2/artifacts.html#structures-resource-profiles),
+        and for requests including all of the required search parameters for each resource type specified in the
+        [C4BB CapabilityStatement](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html) and the
+        [US Core Client CapabilityStatement](https://hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-client.html).
+      )
+
       test from: :initial_wait_test
     end
 
     group do
       title 'C4BB Patient Profile claims data and search request tests'
+      description %(
+        These tests look through all the requests sent in from the client system to ensure a request was made that
+        returned a Patient resource implementing the CARIN for Blue Button [Patient profile](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Patient.html).
+        It will also verify that the client made a request using each required search parameter for the Patient resource
+        specified in the [CARIN IG for Blue Button](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html#patient)
+        and the [US Core Client](https://www.hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-client.html#patient)
+        Capability Statements.
+      )
+
       test from: :patient_claims_data_request_test
       test from: :patient_required_searches
     end
 
     group do
       title 'C4BB Coverage Profile claims data and search request tests'
+      description %(
+        These tests look through all the requests sent in from the client system to ensure a request was made that
+        returned a Coverage resource implementing the CARIN for Blue Button [Coverage profile](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Coverage.html).
+        It will also verify that the client made a request using each required search parameter for the Coverage
+        resource specified in the [CARIN IG for Blue Button](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html#coverage)
+        and the [US Core Client](https://www.hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-client.html)
+        Capability Statements.
+      )
+
       test from: :coverage_claims_data_request_test
       test from: :coverage_required_searches
     end
 
     group do
       title 'C4BB Organization Profile claims data and search request tests'
+      description %(
```

Comment:
Organization, Practitioner, and RelatedPerson are all SHOULD requirements on the [CARIN](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html) and [US Core Client](https://build.fhir.org/ig/HL7/US-Core/CapabilityStatement-us-core-client.html) capabilityStatements.  I think we should make these groups optional.